frontend/src/features/SideWindow/beacon_malfunctions/BeaconMalfunctionDetailsFollowUp.tsx

```diff
@@ -29,9 +30,14 @@ export function BeaconMalfunctionDetailsFollowUp({ beaconMalfunctionWithDetails,
   const [today, setToday] = useState('')
   const [yesterday, setYesterday] = useState('')
   const scrollToRef = useRef<HTMLDivElement>(null)
+  const scrollEventRef = useRef<HTMLDivElement>(null)
```

Comment:
I would have kept the element name because that's what it is: something like `commentsAndActionsWrapperRef`.